debian7/Dockerfile

```diff
@@ -2,10 +2,12 @@ FROM debian/eol:wheezy
 ARG DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update \
-  && apt-get install -y build-essential curl bzip2 libz-dev
+  && apt-get install -y build-essential bzip2 curl libz-dev texinfo
 
 RUN useradd -m -s /bin/bash linuxbrew
 
+ENV CFLAGS="-march=core2"
```

Comment:
Do we want to add an `-O2` or `-Os` in here?

Reply:
Yes, we should probably just use `-O2` to match what we do in homebrew-core.